projects/ROCKNIX/packages/tools/rocknix-abl/package.mk

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2025 ROCKNIX (https://github.com/ROCKNIX)
+
+PKG_NAME="rocknix-abl"
+PKG_VERSION="1.0"
+PKG_ARCH="aarch64"
+PKG_SITE="https://github.com/ROCKNIX/packages"
+PKG_URL="${PKG_SITE}/raw/refs/heads/main/rocknix-abl.tar.gz"
+PKG_DEPENDS_HOST="toolchain"
+PKG_LONGDESC="ROCKNIX ABL."
+PKG_TOOLCHAIN="manual"
```

Comment:
```suggestion
```
This line is not needed as there is nothing to compile, it can install it immediately rather than waiting for toolchain.